Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -254,6 +254,7 @@ open class NavigationView: UIView {
     }
     
     func commonInit() {
+        DayStyle().apply()
```

Comment:
Is it required?

Reply:
This is the easiest way to apply styling for the views in `NavigationView`, as some of them are blank. Users are able to change colors to anything else as well.